### PR TITLE
AVR interrupt based stepper

### PR DESCRIPTION
**Pull request by andre-stefanov**

### CODE REVIEW

File: src/DayTime.cpp [hunk 1]
```diff
@@ -153,14 +153,14 @@ void DayTime::addHours(float deltaHours)
 
 void DayTime::checkHours()
 {
-    while (totalSeconds >= secondsPerDay)
+    while (totalSeconds >= SIDEREAL_SECONDS_PER_DAY)
```
Comment (ClutchplateDude) [inline src/DayTime.cpp]: This must be reverted. This needs to be actual seconds, not sidereal.

**Review by ClutchplateDude — CHANGES_REQUESTED** [review]

File: src/DayTime.cpp (hunk 1 above)
Comment (andre-stefanov) [reply src/DayTime.cpp]: done

**Review by ClutchplateDude — DISMISSED** [review]
Great work! This is going to be amazing!

**Review by ClutchplateDude — APPROVED** [review]
LGTM

**Pull request merged**